### PR TITLE
New package: system76-keyboard-configurator-1.3.10

### PR DESCRIPTION
**Pull request by heppu**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64

### CODE REVIEW

File: srcpkgs/system76-keyboard-configurator/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+version=1.3.10
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+makedepends="gtk+3-devel hidapi-devel eudev-libudev-devel glib-devel"
+short_desc="Tool for configuring System76 keyboards"
+maintainer="Henri Koski <henri.t.koski@gmail.com>"
+license="GPL-3.0-only"
+homepage="https://github.com/pop-os/keyboard-configurator"
+distfiles="https://github.com/pop-os/keyboard-configurator/archive/refs/tags/v1.3.10.tar.gz"
```
Comment (classabbyamp) [inline srcpkgs/system76-keyboard-configurator/template:12]: use `$version` instead of hardcoding it in the url

Comment (heppu) [reply srcpkgs/system76-keyboard-configurator/template]: Fixed

**Comment by heppu** [comment]
@classabbyamp and @icp1994 Does this look now ok?

**Comment by icp1994** [comment]
Yes

**Comment by heppu** [comment]
Any change getting this merged?

File: srcpkgs/system76-keyboard-configurator/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'system76-keyboard-configurator'
+pkgname=system76-keyboard-configurator
+version=1.3.10
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config glib-devel"
```
Comment (classabbyamp) [inline srcpkgs/system76-keyboard-configurator/template:6]: this links against glib-devel, shouldn't it be in makedepends?

Comment (icp1994) [reply srcpkgs/system76-keyboard-configurator/template:6]: Then it should be `libglib-devel` in makedepends I think. You would still need the `glib-devel` in hostmakedepends for cross-compiling as it tries to run `/usr/bin/glib-compile-resources`.

Comment (heppu) [reply srcpkgs/system76-keyboard-configurator/template:6]: Fixed.

**Comment by heppu** [comment]
So any thought on this @classabbyamp @icp1994?

**Comment by heppu** [comment]
ping @classabbyamp @icp1994

**Comment by heppu** [comment]
Is there still something I should do to get this merged @classabbyamp?

**Comment by heppu** [comment]
ping @classabbyamp @icp1994

File: srcpkgs/system76-keyboard-configurator/template (hunk 2 above)
Comment (classabbyamp) [reply srcpkgs/system76-keyboard-configurator/template:6]: did you forget to push the fix?

**Pull request closed without merging**